shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -80,7 +115,7 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "report [category] [category_args]";
+    return "report <category> <category args>";
```

Comment:
the convention is `<required arg>`, `[optional arg]`

Reply:
fixed, thanks